site/content/entry/yuzu-progress-report-jan-2024/index.md

```diff
@@ -0,0 +1,550 @@
+
+Here you can see native 30 FPS vs Lossless Scaling generating frames to 60 FPS videos on `The Legend of Zelda: Breath of the Wild`. 
+Due to upload size and bandwidth limitations from your writer’s ISP, we're using YouTube, which will limit the framerate to 60 FPS, losing detail on the 120 FPS framegen examples.
+
+{{< youtube v-U7GJYrY64 >}} {{< youtube XNBTxr6HBlA >}}
+
+Native 30 FPS vs Lossless Scaling generating frames with a 30 FPS base.
+You can see with so little information, artifacts are common. Lossless Scaling doesn’t handle scene transitions.
+
+{{< youtube CG_e5yOnd9E >}} {{< youtube UpvdLJUtEis >}}
+
+Native 60 FPS vs Lossless Scaling generating frames with a 60 FPS base.
+With more information to play with, Lossless Scaling does a better job. Scene transitions are still an issue.
+
+While the quality is not perfect, it’s a simple and harmless way of improving the experience on any hardware–especially on ~~ugly~~ 30 FPS games like the Pokémon series, or cinematic experiences like the Xenoblade saga. No double standards here.
+Still, for those with recent AMD GPUs (RDNA2 and RDNA3 so far), there’s a better option available now:
+
+### AMD Fluid Motion Frames
+
+Driver release 24.1.1 introduced the first public release of [AFMF](https://community.amd.com/t5/gaming/amd-fluid-motion-frames-is-out-now-on-amd-radeon-rx-7000-series/ba-p/634372), AMD’s game independent frame generation algorithm.
+
+“But writer, AFMF is only for Direct3D 11 and 12 games!” You say.
+That’s where you’re wrong, you see, the [Guru3D forums](https://forums.guru3d.com/threads/amd-software-adrenalin-edition-23-40-01-10-preview-driver-for-amd-fluid-motion-frames.449598/page-35#post-6197794) got some fantastic information, AFMF can be run on Vulkan, OpenGL, and Direct3D 9/10 too, it only needs a registry edit:
+
+```
+[HKEY_LOCAL_MACHINE\SOFTWARE\AMD\DVR]
+"GFGEnableAPI"=dword:00000007
+```
+
+Manually make this change with regedit, reboot Windows if needed, add yuzu to Radeon Software, set Wait for vertical refresh to Disabled, enable AMD Fluid Motion Frames:
+
+{{< imgs
+	"./radeon.png| Good looking UI, AMD"
+  >}}
+
+In the performance tab, you can set the overlay to show the generated frames performance (it won’t show up on regular apps or yuzu, as it isn’t informed by the game engine), along with latency and stuttering.
+
+Open yuzu, set Fullscreen Mode to Exclusive Fullscreen:
+
+{{< imgs
+	"./fullscreen.png| It’s as shrimple as that"
+  >}}
+
+And that’s it, start a game, go into fullscreen by pressing F11 or the hotkey assigned to your controller, and enjoy 2x to 3x the perceived framerate.
+
+Note that while the image quality is much better than the result from Lossless Scaling, AMD disables AFMF if there is too much variance between one frame and the next in order to avoid smoothing out scene transitions like camera changes or opening the menu.
```

Comment:
```suggestion
Note that, while the image quality is much better than the result from Lossless Scaling, AMD disables AFMF if there is too much variance between one frame and the next in order to avoid smoothing out scene transitions like camera changes or opening the menu.
```